superduperdb/base/document.py

```diff
@@ -22,6 +22,7 @@
 ItemType = t.Union[t.Dict[str, t.Any], Encodable, ObjectId]
 
 _OUTPUTS_KEY: str = '_outputs'
+# TODO: Remove this dict to map leaf types to classes
```

Comment:
Why?

Reply:
We can handle this dynamically through automatic import. Otherwise, if we add some special things in the future, we need to add them here.

不过好像并不会太麻烦，好吧，忽略他，我将删除这个comment